README.md

```diff
@@ -181,12 +181,28 @@ This would show a spinning wheel until jenkins is idle. On running locally the h
 <a id="apis"></a>
 ## APIs
 
-This project opens two ports 9091 and 8080. Proxy runs on 8080 and API router runs 9091. 
-The API router has only one API, which is info API. An example is as follows
+This project opens three ports 9091, 9092 and 8080. Proxy runs on 8080. Apart from that we have two API routers one on 9091 and one on 9092. Proxy service running on 8080 is exposed at route https://jenkins.openshift.io and Jenkins API router is exposed on https://jenkins.api.openshift.io. Service running on 9092 is not exposed.
```

Comment:
three ports on....

there is a bit of redudancy between `proxy runs on 8080` and the next part from `Proxy service running on 8080`

You don't explain what is it on 9092 that are not exposed

Reply:
> You don't explain what is it on 9092 that are not exposed
9092 is exposed. 9091 is unexposed.